API/controlsClass.m

```diff
@@ -363,15 +362,16 @@
             if isscalar(obj)
                 dispPropList = masterPropList;
                 if strcmpi(obj.procedure, 'calculate')
-                    dispPropList = rmfield(masterPropList, [deCell, simplexCell, nsCell, dreamCell]);
+                    dispPropList = rmfield(masterPropList, [deCell, simplexCell, nsCell, dreamCell, {'updatePlotFreq','updateFreq'}]);
                 elseif strcmpi(obj.procedure, 'simplex')
                     dispPropList = rmfield(masterPropList, [deCell, nsCell, dreamCell]);
                 elseif strcmpi(obj.procedure, 'de')
                     dispPropList = rmfield(masterPropList, [simplexCell, nsCell, dreamCell]);
+                    % Add the update back...
                 elseif strcmpi(obj.procedure, 'ns')
-                    dispPropList = rmfield(masterPropList, [simplexCell, deCell, dreamCell]);
+                    dispPropList = rmfield(masterPropList, [simplexCell, deCell, dreamCell, {'updatePlotFreq','updateFreq'}]);
                 elseif strcmpi(obj.procedure, 'dream')
-                    dispPropList = rmfield(masterPropList, [simplexCell, deCell, nsCell]);
+                    dispPropList = rmfield(masterPropList, [simplexCell, deCell, nsCell, {'updatePlotFreq','updateFreq'}]);
```

Comment:
The "updatePlotFreq" and "updateFreq" variables already appear in the "masterPropList". Are they supposed to be there twice?